calico-enterprise/network-policy/application-layer-policies/alp.mdx

```diff
@@ -0,0 +1,67 @@
+---
+description: Enforce application layer policies in your cluster to limit access controls based on L7 attributes.
+---
+
+# Enable and enforce application layer policies
+
+Application layer policies let you limit access controls based on L7 attributes. 
+
+## Before you begin
+
+### Unsupported
+{{prodname}} implements application layer policy using Envoy as a DaemonSet. This means you cannot use application layer policy alongside a service mesh like Istio.
+
+### Limitations
+- Application layer policy supports restricting only ingress traffic
+- Support for L7 attributes are limited to HTTP method, and URL exact/prefix path 
+- Supported protocols are limited to TCP-based protocols (for example, HTTP, HTTPS, or gRPC)
+- You can control application layer policies only at the cluster level (not per namespace) 
+
+## How to
+- [Enable application layer policies](#enable-application-layer-policies)
+- [Enforce application layer policies for ingress traffic](#enforce-application-layer-policies-for-ingress-traffic)
+- [Disable application layer policies](#disable-application-layer-policies)
+
+### Enable application layer policies (ALP)
+In the ApplicationLayer custom resource, set the `applicationLayerPolicy` field to Enabled.
+
+```                                                                                           
```

Comment:
```yaml

Reply:
@bmckercher123 I'm unable to see the change here.